lib/purchase_type.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:json_annotation/json_annotation.dart';
+
+/// Supported SKU types.
+enum PurchaseType {
```

Comment:
i'm forgetting why this wouldn't just be `ProductType`? it's not really a type of purchase

Reply:
This is what already excited but let's deprecate this and add a new one! 💪

Reply:
thank you!!